Functions/BouncingNumbers.cs

```diff
@@ -2,7 +2,25 @@
 
 public class BouncingNumbers
 {
-    /*
-	 * TODO: Write helper function for 'BouncingNumbers'
-	 */
+	public static bool IsJumpingNumber(int num)
+	{
+		bool isAsc = true;
+		bool isDesc = true;
+		bool firstDigit = true;
+		int currDigit = 0;
```

Comment:
Maybe set `currDigit` to be `num % 10` and then the loop will be simpler...
Having a condition to differentiate from the first case and the rest is bad practice.